docs/features-timeline.md

```diff
@@ -0,0 +1,252 @@
+  ```js
+  let happy = "wooo!";
+  export { happy as "😃" };
+  ```
+- Parser support for the [Import Assertions](https://github.com/tc39/proposal-import-assertions) proposal
+  ```js
+  import json from "./foo.json" assert { type: "json" };
+  ```
+- [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/) support
+
+</li>
+<li data-date="Jul 2020">
+
+## Babel 7.11.0
+
+[blog post](https://babeljs.io/blog/2020/07/30/7.11.0)
+
+- Enable the [Logical Assignment](https://github.com/tc39/proposal-logical-assignment/) and
+  [Numeric Separator](https://github.com/tc39/proposal-numeric-separator) Stage 4 proposals by default
+- Parser support for the [Decimal](https://github.com/tc39/proposal-decimal) proposal
+  ```js
+  console.assert(0.1m + 0.2m === 0.3m);
+  ```
+- [TypeScript 4.0](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/) support
+
+</li>
+<li class="no-children">
+
+## @babel/eslint-parser
+
+[blog post](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint)
+
+</li>
+<li data-date="May 2020">
+
+## Babel 7.10.0
+
+[blog post](https://babeljs.io/blog/2020/05/25/7.10.0)
+
+- Enable parser support for the [`import.meta`](https://github.com/tc39/proposal-import-meta/) Stage 4 proposal by default
+- Support for the [Ergonomic brand checks for Private Fields](https://github.com/tc39/proposal-private-fields-in-in) proposal
+  ```js
+  class Car {
+    #plate;
+    race(other) {
+       if (#plate in other) console.log("Racing against another car!");
+    }
+  }
+  ```
+
+</li>
+<li class="no-children">
+
+## babel-polyfills
+
+[repository](https://github.com/babel/babel-polyfills)
+
+</li>
+<li data-date="Mar 2020">
+
+## Babel 7.9.0
+
+[blog post](https://babeljs.io/blog/2020/03/16/7.9.0)
+
+- [`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) option in `@babel/preset-env`, to workaround browsers bugs rather than compiling whole Features
+- [TypeScript 3.8](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/) support
+- Support for Flow `declare` class fields
+- Support for the [automatic](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) JSX runtime
+
+</li>
+<li data-date="Jan 2020">
+
+## Babel 7.8.0
+
+[blog post](https://babeljs.io/blog/2020/01/11/7.8.0)
+
+- Enable the [Optional Chaining](https://github.com/tc39/proposal-optional-chaining) and the
+  [Nullish Coalescing](https://github.com/tc39/proposal-nullish-coalescing) Stage 4 proposals by default
+- Support `.mjs` configuration files
+
+</li>
+<li data-date="Nov 2019">
+
+## Babel 7.7.0
+
+[blog post](https://babeljs.io/blog/2019/11/05/7.7.0)
+
+- Parser support for the [top-level `await`](https://github.com/tc39/proposal-top-level-await) proposal
+  ```js
+  import db from "./database.js";
+
+  await db.connect();
+  ```
+- Add error recovery support for Early Errors in `@babel/parser`
```

Comment:
later: Maybe we should have a larger section on errorRecovery to link, or a simple example of the AST or how you might integrate https://babeljs.io/docs/en/babel-parser